README.md

```diff
@@ -35,9 +35,19 @@ lang := enry.GetLanguage("foo.cpp", "<cpp-code>")
 
 Developmemt
 -----------
-*enry* re-uses parts of original [linguist](https://github.com/github/linguist) especially data in `languages.yml` to generate internal data structures. In oreder to update to latest upstream run
+*enry* re-uses parts of original [linguist](https://github.com/github/linguist)  to generate internal data structures. In order to update to latest upstream and generate the necessary code you must run:
 
-    make clean code-generate
+    go generate
```

Comment:
`go generate` runs `make code-generate` that download linguist if it's not downloaded, do we want to run the `clean` rule too and always download linguist?